src/index.html

```diff
@@ -11,6 +11,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big" class="form-1">
+      <input
+      data-qa="keypress"
+      class="search__input-1"
+      type="text"
+      name="search-big"
+      placeholder="Try &quot;Los Angeles&quot;"
```

Comment:
```suggestion
        data-qa="keypress"
        class="search__input-1"
        type="text"
        name="search-big"
        placeholder="Try &quot;Los Angeles&quot;"
```
Не забывай про отступы